### PR TITLE
[frontend] exclusion lists status (#8941)

### PR DESCRIPTION
**Pull request by frapuks**
### Proposed changes

* Display exclusion synchronisation status

### Related issues

* #8941 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

**Comment by Gwendoline-FAVRE-FELIX** [comment]
Functionally it works. All that's missing is a few  adjustments, otherwise everything's fine.

**Comment by frapuks** [comment]
Ready for second review

**Review by Gwendoline-FAVRE-FELIX — APPROVED** [review]

File: opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListsStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+          </Paper>
+        </Grid>
+        <Grid item xs={4}>
+          <Paper
+            variant="outlined"
+            style={{ display: 'flex', padding: 20, height: 100, position: 'relative' }}
+            className={'paper-for-grid'}
+          >
+            <UpdateOutlined color="primary" style={{ fontSize: 40, position: 'absolute', top: 25, right: 15 }} />
+            <div>
```
Comment (ValentinBouzinFiligran) [inline opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListsStatus.tsx:91]: is this div useful ? 
can we replace it by <> ?

Comment (frapuks) [reply opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListsStatus.tsx:91]: It is usefull to have 1 block for flexbox.
If I replace it by <>, flexbox considered the children of this div as children of the paper, and align horizontally these 2 texts with the icon.

**Review by ValentinBouzinFiligran — APPROVED** [review]

**Pull request merged**